File: Tl2.4 RESTFULL_DOTNET_EUREKA_BANK_GR08/01.SERVIDOR/ws_eureka_bank_restful_dotnet/ec.edu.monster.model/OperacionResponse.cs
namespace ec.edu.monster.model;

public class OperacionResponse
{
    public bool Success { get; set; }
    public double Saldo { get; set; }
    public string? Mensaje { get; set; }

    public OperacionResponse()
    {
    }

    public OperacionResponse(bool success, double saldo, string? mensaje)
    {
        Success = success;
        Saldo = saldo;
        Mensaje = mensaje;
    }
}

File: Tl2.4 RESTFULL_DOTNET_EUREKA_BANK_GR08/02.CLIMOV/app/src/main/java/ec/edu/climov_eureka_gr08/model/OperacionResponse.java
package ec.edu.climov_eureka_gr08.model;

public class OperacionResponse {
    private boolean success;
    private double saldo;
    private String mensaje;

    public OperacionResponse() {
    }

    public boolean isSuccess() {
        return success;
    }

    public void setSuccess(boolean success) {
        this.success = success;
    }

    public double getSaldo() {
        return saldo;
    }

    public void setSaldo(double saldo) {
        this.saldo = saldo;
    }

    public String getMensaje() {
        return mensaje;
    }

    public void setMensaje(String mensaje) {
        this.mensaje = mensaje;
    }
}
